libs/dml/src/relation_info.rs

```diff
@@ -57,6 +57,8 @@ pub enum ReferentialAction {
     /// deferred, this makes it possible to temporarily violate integrity in a
     /// transaction while making sure that subsequent operations in the
     /// transaction restore integrity.
+    /// When using referential integrity with relation mode "prisma", NoAction
+    /// becomes an alias of the simulated Restrict when supported.
```

Comment:
```suggestion
    /// When using relation mode "prisma", NoAction becomes an alias of the simulated 
    /// Restrict when supported.
```

Reply:
```suggestion
    /// When using referential integrity with relation mode "prisma", NoAction
    /// becomes an alias of the emulated Restrict when supported.
```

Nit: we mention `emulated` / `emulation` for this functionality, using the same term everywhere could be nice

Reply:
I'll integrate this with @Jolg42's suggestion in the [other PR](https://github.com/prisma/prisma-engines/pull/3274#discussion_r991919660): 

```suggestion
    /// When using relationMode = "prisma", NoAction becomes an alias of
    /// the emulated Restrict (when supported).
```